documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -0,0 +1,474 @@
+
+* for an alpha release, the delta with respect to the previous release
+* for the first release candidate, all changes since the last public release
+* for subsequent release candidate(s), only the delta to the previous release candidate
+* for a public release, the consolidated changes since the previous public release
+
+# r2.6 - patch release
+
+## Release Notes
+
+* quality-on-demand v0.10.1
+* Commonalities v0.2.0
+* ICM v0.1.0
+* Backward compatible: yes
+
+<!--For each above listed API version that changed, create the following section, replacing the API-na,e and API-version-x.y.z with actual API andme and version-->
+
+**This release contains a patch for v0.10.0 of the Quality-On-Demand (QoD) API. Please read also the notes and changes for r1.6**
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.1/code/API_definitions/qod-api.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.1/code/API_definitions/qod-api.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/v0.10.1/code/API_definitions/qod-api.yaml)
+
+## Changes for quality-on-demand v0.10.1
+
+### Added
+
+* Added configuration for linting ruleset by @rartych in https://github.com/camaraproject/QualityOnDemand/pull/270
+
+Other examples
+
+* New property `new_name`
+* New endpoint `new_name`
+* New test case feature files
+* Added examples
+
+### Changed
+
+* Updated the documentation to address the lack of `statusInfo` in `SessionInfo` temporary by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/269
+  * Note: The parameter `statusInfo` will be added to `SessionInfo` within next regular release
+* Updated the project scope in the README.md by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/255
+
+Other examples
+
+* Property `old_name` renamed to `new_name`
+* Format for property `property` changed to `new_format`
+
+### Fixed
+
+* Fixed maximum duration in session info and improved documentation by @emil-cheung in https://github.com/camaraproject/QualityOnDemand/pull/277
+  * Improved the documentation of "Extend the duration of an active session"
+  * Improved the datatype "SessionInfo" to remove the maximum limit of duration
+
+Other examples
+
+* Typos fixed
+* Reference corrected
+
+### Removed
+
+Examples:
+
+* Deprecated field `old_field`
+* Deprecated endpoint
+
+# r2.5 - initial public release
+
+## Release Notes
+
+* quality-on-demand v0.10.0
+* Commonalities v0.2.0
+* ICM v0.1.0
+* Backward compatible wrt v0.9.0 ([r1.3](#r13---initial-public-release)): **no**
+
+**This release contains the initial public version of the v0.10.0 of the Quality-On-Demand (QoD) API.**
+
+- **This release contains significant changes compared to r1.5, and is not backward compatible**
```

Comment:
```suggestion
* This release contains the definition and documentation of
  * quality-on-demand v0.10.0
* The API definition(s) are based on
  * Commonalities v0.2.0
  * Identity and Consent Management 0.1.0

## quality-on-demand v0.10.0

**quality-on-demand v0.10.0 is an initial public version with significant changes compared to the previous initial version v0.9.0 and is not backward compatible** 
```

Hint: API version should be compared with API versions, not API version with releases!

Reply:
Updated as suggested